### ISSUE TITLE
Update `api_gui.md` for Berkeley schema and deploy to documentation website

### ISSUE THREAD

**Issue by aclum** [microbiomedata/NMDC_documentation#65]
In particular the mentions to Activity should be removed in favor of WorkflowExecution. Depends on [updates to the activity endpoints](https://github.com/microbiomedata/nmdc-runtime/issues/593). It would be good to start a branch for berkeley so we can have the nmdc-runtime's berkeley branch point to a branch in NMDC_documentation that contains the updated text.

**Comment by brynnz22** [comment]
Sounds good. I'll work on this. Thanks!

**Comment by mslarae13** [comment]
@brynnz22 can you add why this is blocked & what needs to happen or decisions that need made to unblock?

**Comment by brynnz22** [comment]
@mslarae13 I don't think this is blocked. It depends on that other PR Alicia linked, but I can start it.

**Comment by aclum** [comment]
The endpoints that reference 'Activity' need to have finalized new names, see hyperlinked ticket in the description.

**Comment by brynnz22** [comment]
Linking this new issue: https://github.com/microbiomedata/nmdc-runtime/issues/653. While updating the docs, realized the old activity endpoints allowed a user to search both `WorkflowExecution` and `PlannedProcess` which allowed searching `OmicsProcessing`(now `DataGeneration` and `MaterialProcessing` collections). Will wait to update API docs based on this issue.

**Comment by mslarae13** [comment]
Thanks @brynnz22 so 
Criteria for unblock, 
- merge in https://github.com/microbiomedata/nmdc-runtime/issues/646
- Complete https://github.com/microbiomedata/nmdc-runtime/issues/653

**Comment by brynnz22** [comment]
Just your second bullet point is the criteria for unblock. The first bullet point should come after I complete this issue.

**Comment by eecavanna** [comment]
Related: https://github.com/microbiomedata/NMDC_documentation/pull/70

**Comment by mslarae13** [comment]
is this post refactor rollout eligible? (AKA, not MVP?)

**Comment by eecavanna** [comment]
Hi @mslarae13, I don't think so. The content updates are done (in PR https://github.com/microbiomedata/NMDC_documentation/pull/70) — they just hasn't been deployed yet. We're deliberately waiting until "switchover" day before deploying them.

**Comment by ssarrafan** [comment]
@mslarae13 @brynnz22 since[ #653](https://github.com/microbiomedata/nmdc-runtime/issues/653) is closed can this issue be closed too?

**Comment by brynnz22** [comment]
No its not done until its merged, and it can't be merged in until after the Berkeley stuff. I put it in the post roll out bin.

**Comment by ssarrafan** [comment]
I'm moving this to the next sprint, pending Berkeley rollout mid-Oct.

**Comment by ssarrafan** [comment]
Berkeley was rolled out, but @brynnz22 is out today and tomorrow so I'll move this to the next sprint to close.

**Comment by eecavanna** [comment]
This issue can be closed. I merged the associated changes into the `main` branch during Tuesday's release.

**Comment by eecavanna** [comment]
Looks like GitHub didn't automatically close this issue because the PR I merged was not linked to it. I will link it shortly.

**Comment by eecavanna** [comment]
I have linked the PR to this issue. Since the merging of the PR has already takes place, GitHub won't automatically close this issue. So, I will close it manually now.
